mds/authent/migrations/0003_auto_20190903_1529.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.1.11 on 2019-09-03 15:29
```

Comment:
You should remove this migration from your code : you didn't change `mds/authent/models.py` file.
As @bors-ltd said, always specify the app name when using `./manage.py migrate` to prevent django from creating migrations on the project's dependencies